packages/http/src/mocker/generator/JSONSchema.ts

```diff
@@ -20,8 +20,15 @@ jsf.option({
   maxLength: 100,
 });
 
+// TODO: adjust if https://github.com/json-schema-faker/json-schema-faker/pull/526 is resolved
+const uuidNotAsURN = (fake: string) => {
+  return fake.replace('urn:uuid:', '');
+};
+
 export function generate(source: JSONSchema): unknown {
-  return jsf.generate(cloneDeep(source));
+  const fake = jsf.generate(cloneDeep(source));
+
+  return source.type === 'string' && source.format === 'uuid' ? uuidNotAsURN(fake) : fake;
```

Comment:
I guess it should work with `uuid`s inside objects too. You probably need to introduce recurrence here.